packages/common/src/core/types/IHubProject.ts

```diff
@@ -23,6 +23,10 @@ export interface IHubProject
    * Project Status
    */
   status: PROJECT_STATUSES;
+  /**
+   * Project display properties
+   */
+  view?: IHubProjectView;
```

Comment:
Since this will be a common property w/ (at least) initiatives... could we generalize this?